pyanaconda/modules/storage/devicetree/devicetree_interface.py

```diff
@@ -28,4 +29,18 @@
 @dbus_class
 class DeviceTreeInterface(DeviceTreeViewerInterface, DeviceTreeHandlerInterface):
     """DBus interface for the device tree module."""
-    pass
+
+    _tree_counter = 1
+
+    @staticmethod
+    def get_object_path(namespace):
+        """Get the unique object path in the given namespace.
+
+        This method is not thread safe for now.
+
+        :param namespace: a sequence of names
+        :return: a DBus path of a device tree
+        """
+        tree_number = DeviceTreeInterface._tree_counter
+        DeviceTreeInterface._tree_counter += 1
+        return get_dbus_path(*namespace, "DeviceTree", str(tree_number))
```

Comment:
Wouldn't it be better to make this method classmethod and use cls variable?

Reply:
No, because every subclass would have its own counter and that could cause name collisions.

Reply:
Fair point, thanks for explanation.